modules/ServerAPI/src/server/Bootstrap.coffee

```diff
@@ -16,15 +18,28 @@ exports.main = (callback) ->
                 callback(error)
             else
                 callback(null)
+    if config.all.server.migrateCmpdRegBulkLoaderFilesToSubfoldersOnStart? && config.all.server.migrateCmpdRegBulkLoaderFilesToSubfoldersOnStart
```

Comment:
This is the code for the bootstrap script to call the migration to move cmpdregfiles to their new locations.